cdap-api/src/main/java/co/cask/cdap/api/dataset/table/Tables.java

```diff
@@ -89,7 +89,7 @@ public static void createTable(ApplicationConfigurer configurer, String datasetN
   /**
    * Creates properties for {@link Table} or {@link Table} data set instance.
    * @param level level on which to detect conflicts in changes made by different transactions
-   * @param ttl time to live for data written into a table, in ms. {@link #NO_TTL} means unlimited
+   * @param ttl time to live for data written into a table, in seconds. {@link #NO_TTL} means unlimited
```

Comment:
So this actually is a non-backward compatible API change. Any old code being deploy again on 3.3 will experience 1000 times longer TTL.